### PR TITLE
House branch

### PR DESCRIPTION
**Pull request by allison-grothman**
To-dos implemented:
- Player has the option of not purchasing a house
- House cards display their price, but currency still hasn't been implemented

### CODE REVIEW

**Comment by taylor-noel** [comment]
Since you added the slide option of not buying a house I don't think we need the initial question. Maybe we can change the initial question to Buy or Sell a house

**Comment by taylor-noel** [comment]
I think my pr for the skills tile might help with the sell option for this. We can use the respin state to handle if the player wants to sell

**Comment by allison-grothman** [comment]
Yeah the respin functionality in your skills pr looks useful for this. I'll work on the sell a house option now, and I'll reopen this pr when I've done that

**Comment by allison-grothman** [comment]
New implemented:
- Player has the option to sell a house
To-do:
- Add calls to Flask endpoints to update player assets (houses and cash)

File: game-app/src/Components/GameBoard.js
Comment (taylor-noel) [inline game-app/src/Components/GameBoard.js:1]: can you please change this back to increment

**Comment by taylor-noel** [comment]
I didn't get the option to sell the house... when is it supposed to show?

File: game-app/src/Components/GameBoard.js
Comment (allison-grothman) [reply game-app/src/Components/GameBoard.js:1]: Done

**Comment by allison-grothman** [comment]
That's strange - it works for me. When you land on a house tile, it asks whether you want to buy or sell a house. Could you try pulling again? Maybe you're missing commits

**Pull request merged**